chromadb/test/property/test_cross_version_persist.py

```diff
@@ -164,18 +167,18 @@ def switch_to_version(version):
 
 
 def persist_generated_data_with_old_version(
-    version,
-    settings,
+    version: str,
+    settings: Settings,
     collection_strategy: strategies.Collection,
     embeddings_strategy: strategies.RecordSet,
-):
+) -> None:
     old_module = switch_to_version(version)
     api: API = old_module.Client(settings)
     api.reset()
     coll = api.create_collection(
         name=collection_strategy.name,
         metadata=collection_strategy.metadata,
-        embedding_function=lambda x: None,
+        embedding_function=strategies.not_implemented_embedding_function(),
```

Comment:
Lets leave this as lambda x: None since this runs in an older version of chroma and we don't want to depend on new things